src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
     set_target_properties (appleseed.cli PROPERTIES
         INSTALL_RPATH "\$ORIGIN/../lib"
     )
-endif ()
+endif (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
Yes. If you make a new commit and push it (to the same branch), it will appear here in the pull request. It's magical :)